plugin-animation/src/main/java/com/mapbox/maps/plugin/animation/CameraAnimatorsFactory.kt

```diff
@@ -133,7 +126,7 @@ class CameraAnimatorsFactory internal constructor(mapDelegateProvider: MapDelega
    * @return Array of the created animators
    */
   internal fun getPitchBy(pitch: Double): Array<CameraAnimator<*>> {
-    mapCameraDelegate.getCameraOptions().pitch?.let { startPitch ->
+    mapCameraDelegate.getCameraState().pitch.let { startPitch ->
```

Comment:
`.let` shouldn't be needed anymore, right?

Reply:
Yep, using `.let` and `?.let` is also fine but we could get rid of it in this case.
Thanks 👍